TEAM-VALUES.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Knative team values"
+linkTitle: "Team values"
+weight: 50
+type: "docs"
+---
+
+We want to make sure every member has a shared understanding of the goals and
+values we hold as a team:
+
+- Optimize for the **overall project**, not your own area or feature
+
+  - A shortcut for one individual can mean a lot of extra work or disruption for
+    the rest of the team.
+
+- Our repos should always be in release shape: **Always Green**
```

Comment:
It would probably be useful to distinguish between `knative` and `knative-community` (which I use to bucket things like `eventing-contrib`, or third-party integrations)

Reply:
Can we do that in another PR? I am just moving that content in this PR.

Reply:
yep, just on the brain :)